src/components/card/card-row/card-row.spec.tsx

```diff
@@ -19,22 +22,29 @@ describe("CardRow", () => {
   });
 
   describe("when styled system padding props are set", () => {
-    testStyledSystemPadding((props) => <CardRow {...props}>Test</CardRow>, {
-      py: "var(--spacing300)",
-    });
+    testStyledSystemPadding(
+      (props) => <CardRow {...props}>Test</CardRow>,
+      {
+        py: "var(--spacing300)",
+      },
+      undefined,
+      { modifier: "&&" }
+    );
   });
 
-  it.each<[Exclude<CardRowProps["spacing"], undefined>, string]>([
+  it.each<[Exclude<CardContextProps["spacing"], undefined>, string]>([
```

Comment:
**nitpick**: would it be worth renaming the test description slightly to reflect that `spacing` is passed via context inside of a prop?

Reply:
I'm not convinced it matters here as it will be set via prop in the parent